scripts/untombstone.py

```diff
@@ -0,0 +1,150 @@
+import os, re
+import logging as logger
+
+from dss import Config, Replica, dynamodb
+from dss.dynamodb import get_item, put_item, delete_item, DynamoDBItemNotFound
+from dss.index.es import ElasticsearchClient
+from cloud_blobstore import BlobNotFoundError
+from dss.index.es.backend.ElasticsearchIndexBackend import index_bundle
+from dss import DSSException, dss_handler, DSSForbiddenException
+from dss.collections import owner_lookup
+
+from hca.dss import DSSClient
+from hca.util.exceptions import SwaggerAPIException
+from dss.storage.identifiers import DSS_BUNDLE_TOMBSTONE_REGEX as dead_template
+
+
+# --------------------------------------------------------------
+# untombstone bundles
+# --------------------------------------------------------------
+
+
+def untombstone_bundle(uuid, replica, version=None):
+    """
+    deletes dead bundle and brings back original bundle 
+    """
+    if version is not None:
+        key = f"{uuid}.{version}.dead"
+    else:
+        key = f"{uuid}.dead"
+    if tombstoned_or_not_bundle(key, replica) is False:
+        pass
+    else:
+        deindex_dead_bundle(key)
+        update_og_bundle(key)
+        # indexs and notifys subs
+        index_bundle(key)
```

Comment:
This should not be called directly. Instead, the normal DSS indexing machinery should be invoked by sending an SQS message to the index queue. You can use [this DSS operations snippet](https://github.com/HumanCellAtlas/data-store/blob/master/dss/operations/elasticsearch.py#L37-L39) as an example.